src/header.tsx

```diff
@@ -155,7 +155,11 @@ export const MainSectionHeader: React.FC<MainSectionHeaderProps> = ({title, desc
 type HeaderLayoutProps = {
     isInverse?: boolean;
     breadcrumbs?: RendersNullableElement<typeof NavigationBreadcrumbs>;
-    header: React.ReactNode; // intentionally not forced to RendersElement<typeof Header> to allow skeletons for example
+    /**
+     * Intentionally not forced to RendersElement<typeof Header> to allow skeletons for example
+     * The header is optional in order to allow webviews to delegate the header visualization to the surrounding native app.
```

Comment:
Please, include this same explanation in the story. Stories should work as documentation, so it's a good place to explain this case

Reply:
done ;)

Reply:
I meant in the html, for the message to appear in the story